README.md

```diff
@@ -194,13 +194,22 @@ you can retrieve it by passing a symbol.
 
 ```ruby
 # app/policies/dashboard_policy.rb
-class DashboardPolicy < Struct.new(:user, :dashboard)
-  # ...
+class DashboardPolicy
+  attr_reader :user
+
+  # _record in this example will just be :dashboard
+  def initialize(user, _record)
+    @user = user
+  end
+
+  def show?
+    user.admin?
+  end
 end
 ```
 
 Note that the headless policy still needs to accept two arguments. The
-second argument will just be the symbol `:dashboard` in this case which
+first argument will just be the symbol `:dashboard` in this case which
```

Comment:
The `_record` is still `:dashboard` :)

Reply:
I believe the first argument provided to `authorize` in this case would be `:dashboard`, whereas the second parameter in the policy's initializer method would in this case be `:dashboard`.

Reply:
Yup! The passage does talk about the headless policy, though?

Reply:
Is this an attempt to fix an unrelated issue? 

In this case the `authorize` method accepts only one argument, while the policy initializer accepts two. This line is pretty clearly talking about the policy initializer since the line before says: 

> Note that the headless policy still needs to accept two arguments. 

Maybe this area of the documentation could be clarified, but that feels completely independent of the change to remove Struct.

Reply:
Changed: 67e0dd3417ee751ccbe1443b0c62e074e6ae2976